packages/expo-print/src/Print.types.ts

```diff
@@ -63,6 +63,11 @@ export type FilePrintOptions = {
    * HTML string to print into PDF file.
    */
   html?: string;
+  /**
+   * **Available on iOS only.** Alternative to `html` option that uses [UIMarkupTextPrintFormatter](https://developer.apple.com/documentation/uikit/uimarkuptextprintformatter)
+   * instead of WebView.
```

Comment:
Perhaps it's useful to clarify that `markupFormatterIOS` has a precedence over `html` option 🤔 I can imagine people wanting to use `html` on Android and `markupFormatterIOS` on iOS — in this case, if it has a precedence, you can safely call the function with both options provided, without checking on which platform it's being run.

This also makes me think that it might be better to have this option as a `boolean`, kind of you're opting in to use `html` with markup formatter.